comfy_api_nodes/nodes_ideogram.py

```diff
@@ -540,7 +518,8 @@ async def api_call(
                     color_palette=color_palette if color_palette else None,
                 )
             ),
-            auth_kwargs=kwargs,
+            auth_token=cls.hidden.auth_token_comfy_org,
```

Comment:
maybe it would be better to use `auth_kwargs` here?

I would remove the `auth_token` and `comfy_api_key` parameters everywhere at the ned, and make only one `auth_kwargs`.

it will be more easily extensible/changeable in the future - if necessary, we can make it a Typed Dict, so that linters would work on it.

WDYT?

Reply:
I changed the code to use `auth_kwargs` as it will be more correct if we want to extend the authentication functionality in the future.